### PR TITLE
apps/v1beta1 -> apps/v1

### PR DESCRIPTION
**Pull request by briandealwis**
Change K8s deployments to use `apps/v1`.

### CODE REVIEW

**Comment by etanshaul** [comment]
Thanks @briandealwis 

@simonz130 ptal

**Comment by etanshaul** [comment]
@briandealwis looks like with this change I am getting the following

```
 error validating data: ValidationError(Deployment.spec): missing required field "selector" in io.k8s.api.apps.v1.Deployment
```

I guess we need to make this update as well for this to work properly

**Comment by etanshaul** [comment]
```
, stderr: error validating "/Users/eshaul/dev/samples/testport/cloud-code-samples/java/java-guestbook/kubernetes-manifests/guestbook-backend.deployment.yaml": error validating data: ValidationError(Deployment.spec): missing required field "selector" in io.k8s.api.apps.v1.DeploymentSpec; if you choose to ignore these errors, turn validation off with --validate=false
error validating "/Users/eshaul/dev/samples/testport/cloud-code-samples/java/java-guestbook/kubernetes-manifests/guestbook-frontend.deployment.yaml": error validating data: ValidationError(Deployment.spec): missing required field "selector" in io.k8s.api.apps.v1.DeploymentSpec; if you choose to ignore these errors, turn validation off with --validate=false
, err: exit status 1: exit status 1 
```

**Comment by etanshaul** [comment]
sorry - ignore my above comment. I was manually updating the version without adding the selector (which @briandealwis clearly already did in this PR).

File: dotnet/dotnet-guestbook/kubernetes-manifests/guestbook-backend.deployment.yaml [hunk 1]
```diff
@@ -30,4 +34,4 @@ spec:
         - name: PORT
           value: "8080"
         - name: GUESTBOOK_DB_ADDR
-          value: dotnet-guestbook-mongodb:27017
\ No newline at end of file
+          value: dotnet-guestbook-mongodb:27017
```
Comment (simonz130) [inline dotnet/dotnet-guestbook/kubernetes-manifests/guestbook-backend.deployment.yaml]: what is the change here?

File: dotnet/dotnet-guestbook/kubernetes-manifests/guestbook-backend.deployment.yaml [hunk 2]
```diff
@@ -12,6 +12,10 @@ metadata:
     tier: backend
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      app: dotnet-guestbook
```
Comment (simonz130) [inline dotnet/dotnet-guestbook/kubernetes-manifests/guestbook-backend.deployment.yaml]: Have you tested deploying the sample after adding this?

File: dotnet/dotnet-guestbook/kubernetes-manifests/guestbook-frontend.deployment.yaml [hunk 3]
```diff
@@ -12,6 +12,10 @@ metadata:
     tier: frontend
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      app: dotnet-guestbook
+      tier: frontend
```
Comment (simonz130) [inline dotnet/dotnet-guestbook/kubernetes-manifests/guestbook-frontend.deployment.yaml]: ditto

File: python/python-guestbook/kubernetes-manifests/guestbook-frontend.deployment.yaml [hunk 4]
```diff
@@ -7,6 +7,10 @@ metadata:
     tier: frontend
 spec:
   replicas: 1
+  selector:
```
Comment (simonz130) [inline python/python-guestbook/kubernetes-manifests/guestbook-frontend.deployment.yaml]: Will it trigger installing newest skaffold for all the folks?
Do we need to do it manually every time there is a new version of skaffold?
What happens if we don't do it and folks still update to newest skaffold? Is it all backward compatible?

File: dotnet/dotnet-guestbook/kubernetes-manifests/guestbook-backend.deployment.yaml (hunk 1 above)
Comment (briandealwis) [reply dotnet/dotnet-guestbook/kubernetes-manifests/guestbook-backend.deployment.yaml]: It looks like vim added a newline to the last line.

**Comment by etanshaul** [comment]
@simonz130 can we get this looked at?

**Comment by etanshaul** [comment]
@simonz130 also to point out: without this change, the Java debugging support in the upcoming 19.5.1 intellij release (coming in the next day or two) one work out of the box with this repo.

**Pull request merged**